metabase/metabase
# Render children of DebouncedFrame only when width is specified

Closes https://github.com/metabase/metabase/issues/44495

### Description

It prevents blinking during QB loading - initially we rendered an absolute container with width 0 in the top left corner which caused a blink

### How to verify

- go to the saved question 
- refresh the page
- check top left corner - there shouldn't be rendered a part of a button there

### Demo

v49 (correct)

https://github.com/metabase/metabase/assets/125459446/817bb754-8cf8-4db1-965b-1da3e3ec2f46

with current fix


https://github.com/metabase/metabase/assets/125459446/76503c17-1206-4d48-817e-5cb0aa0993a9

### Checklist

- I doubt we need a test for it, the issue most likely was caused by css migration to css modules
